### ISSUE TITLE
Add a flag for BDR, WB, BD, DVD, TV, HDTV, source of video file.

### ISSUE THREAD

**Issue by felisucoibi** [cellar-wg/matroska-specification#365]
This is the source of the video file information. Now more or less you can guess with bit-rate, and watching the video but could be nice to have it in a flag. All these is useful for archiving mkvs, like i'm doing for historical reasons and preservation.

UHDR, is a untouched video from a UHD bluray.
UHD, is a 4k file source is a BD but video compressed.
BDR, is a untouched video from a bluray, bd remux
WB, is downloaded form a streaming website.
BD, bd, is a bd source video but compressed.
DVDR, is an uncomrpess dvd video file, dvd remux.
DVD, is a dvd video source
VHS, is a vhs rip
TV, is a captures tv show
HDTV, is a captures hd tv show.

**Comment by hubblec4** [comment]
You could set this in the name of a track.
When I preserve my BDs then I set such quality info about the video in the name of the video stream.

mostly: h264 untouched

Alternative you can use Matroska Tags. It is easy to use.

This issue is close the same like this one
https://github.com/cellar-wg/matroska-specification/issues/357 (opened by you)

**Comment by felisucoibi** [comment]
My goal with all thes proposed changes is to standarize the real world of mkv files on internet. I have other proposals for movie information that can be achieved just adding some txt files inside the matroska but it needs to be standarized.

**Comment by hubblec4** [comment]
I can understand you and it's a long way.
Extra info could be added as attachment.
For me are the Tags a powerful feature. You can do all what you want(and more).

All things which essential for playback will be implemented directly in the stream level/section.
All other info could/should uses Tags.

**Comment by robUx4** [comment]
There's already a `ORIGINAL_MEDIA_TYPE` tag for that. There's no set/standardized value in the current spec. There's also mention of the [ID3v2 TMED tag](http://id3.org/id3v2.3.0#TMED).

**Comment by felisucoibi** [comment]
> There's already a `ORIGINAL_MEDIA_TYPE` tag for that. There's no set/standardized value in the current spec. There's also mention of the [ID3v2 TMED tag](http://id3.org/id3v2.3.0#TMED).

The id3c2 tag is very outdated.

What i'm proposing is based in 150.000 media files I tested, these are more or less all the possibilities. Maybe more people can review and suggest , but i think they are all.

**Comment by robUx4** [comment]
Yes, we can add more values to `ORIGINAL_MEDIA_TYPE` it doesn't have to stick to ID3v2 since it's our own tag.

I think the tag name is OK because it mentions the original media type without claiming it's an exact replica. For example you could reduce the bitrate in half, use another codec, remove some tracks but it's still from that original media.

**Comment by robUx4** [comment]
Closing this as we have the appropriate tag (we just can't tell what modification are done from the original media). We won't standardize all the possible values. Or it should be a separate issue.
